contracts/SwapKiwiV2.sol

```diff
@@ -0,0 +1,320 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.1;
+
+import "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
+import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract SwapKiwiV2 is Ownable, ERC721Holder, ERC1155Holder {
```

Comment:
also could you rename it back to SwapKiwi (without v2) as it messes up stuff on frontend and in bot.

Reply:
I will add the comment.
And if renaming it back to SwapKiwi, there are 2 contracts which have the same name in folder so it can cause a problem in the test.

Reply:
Lets rename old one to SwapKiwiV1.

Reply:
yeah